packages/client/src/context/Auth.context.tsx

```diff
@@ -106,28 +115,96 @@ export const AuthProvider: FC<AuthProviderProps> = ({ children }) => {
     handleUnauthenticated();
   };
 
+  // Handle switch tab
+  const handleTabChange = (_event: React.SyntheticEvent, tab: 'login' | 'signup' | 'reset') => {
+    setActiveTab(tab);
+  };
+
+  // Handle organization select
+  const handleOrganizationSelect = (event: SelectChangeEvent<string>) => {
+    const selectedOrganization = organizationList.find((org) => org.name === event.target.value);
+    setOrganization(selectedOrganization || null);
+  };
+
   return (
-    <AuthContext.Provider value={{ token, authenticated, decodedToken, logout }}>
-      {!authenticated && organization && (
-        <FirebaseLoginWrapper setToken={handleAuthenticated} organization={organization} />
-      )}
-      {authenticated && children}
-    </AuthContext.Provider>
+    <Container
+      maxWidth="sm"
+      sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center', height: '80vh' }}
```

Comment:
I see this used in a few places, you probably want to checkout the [MUI Stack](https://mui.com/material-ui/react-stack/) component for column oriented views